pages/linux/numfmt.md

```diff
@@ -0,0 +1,15 @@
+# numfmt
+
+> Convert numbers from/to human-readable strings.
```

Comment:
Perhaps `to or from` reads better.

Reply:
Or `to and from`?

Reply:
That change is all set, I went with "to and from" because that has the right "sound" to my ears. Thanks all.